services/meta/data.go

```diff
@@ -1363,12 +1363,15 @@ func (sgi ShardGroupInfo) clone() ShardGroupInfo {
 	return other
 }
 
-// ShardFor returns the ShardInfo for a Point.
-func (sgi *ShardGroupInfo) ShardFor(p models.Point) ShardInfo {
+type hashIDer interface {
+	HashID() uint64
+}
+
+// ShardFor returns the ShardInfo for a Point or other hashIDer.
+func (sgi *ShardGroupInfo) ShardFor(p hashIDer) ShardInfo {
```

Comment:
`hashIDer` is a much easier requirement than `Point` (an interface subset)

Reply:
Wouldn't have thought of this, I like it!

Reply:
Since `ShardFor` is public, `hashIDer` should probably be public also?

Reply:
The interface only needs to be exported if we want to re-use it elsewhere, which we don't (I think), and also this package would not be the right one to export from.

I have validated locally that I can use this to fix the clustering build.